packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol

```diff
@@ -186,6 +185,31 @@ library ForgeArtifacts {
         });
     }
 
+    /// @notice Returns the storage slot for a given contract and slot name
```

Comment:
It would be reasonable to ask  why I did not reafctor the function above to use this new function as the code is mostly redundant. 

The difference is that this code does not include the `slotType` in the `jq  select()` function. That is IMO not necessary here, as `_slotNames` should not be duplicable within a layout, and the type names are often unintuitive. 

For example, in the `OptimismPortal2` layout, the `superchainConfig` entry is the following:

```
    {
      "astId": 81707,
      "contract": "src/L1/OptimismPortal2.sol:OptimismPortal2",
      "label": "superchainConfig",
      "offset": 1,
      "slot": "53",
      "type": "t_contract(ISuperchainConfig)1392"
    }
```